src/desktop/components/react/stitch_components/ReactionArtworkDetails.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+import { Theme } from "@artsy/palette"
+import { ContextProvider } from "reaction/Artsy"
+import { ArtworkDetailsQueryRenderer as Renderer } from "reaction/Apps/Artwork/Components/ArtworkDetails"
+import { data as sd } from "sharify"
+
+export const ReactionArtworkDetails = props => {
+  return (
+    <Theme>
```

Comment:
@oxaudo - For style stuff, can wrap component in a `<Theme>` from `@artsy/palette`

Reply:
Oh... my bad with `.js` file and thanks for the `<Theme> addition! Seems pretty straightforward.